src/game/shared/schemainitutils.h

```diff
@@ -25,14 +25,17 @@
 		}																\
 		else															\
 		{																\
-#ifndef VALVE_PURE														\
+/* note(replaycoding):
+ * this ifndef is invalid syntax, according to
+ * https://stackoverflow.com/questions/10074520/why-compiler-complain-about-this-macro-declaration */ \
+/* #ifndef VALVE_PURE														*/ \
```

Comment:
This needs to be removed, or if you really can't figure it out, then remove the whole commented out part.